### PR TITLE
fix(*): repair delimiter in repo, repair install variable, trap exits, fix make install + uninstall

### PR DESCRIPTION
**Pull request by oklopfer**

### CODE REVIEW

File: pacstall-qa [hunk 1]
```diff
@@ -30,11 +30,12 @@ def main [
     }
     let parsed_pr = (parse-pr $metalink)
     let provider_url = (parse-link $parsed_pr $install)
-    print $"Replacing '/usr/share/pacstall/repo/pacstallrepo'. Do not cancel this action!!!"
+    print $"Backing up '/usr/share/pacstall/repo/pacstallrepo'"
     ^sudo mv /usr/share/pacstall/repo/pacstallrepo /usr/share/pacstall/repo/pacstallrepo.pacstall-qa.bak
-    $provider_url.0 | ^sudo tee /usr/share/pacstall/repo/pacstallrepo
+    ^echo $"($provider_url.0)" | ^sudo tee /usr/share/pacstall/repo/pacstallrepo out> /dev/null
```
Comment (Elsie19) [inline pacstall-qa]:
```suggestion
    print $"($provider_url.0)" | ^sudo tee /usr/share/pacstall/repo/pacstallrepo out> /dev/null
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: pacstall-qa (hunk 1 above)
Comment (oklopfer) [reply pacstall-qa]: wont work silly

File: pacstall-qa.wrapper [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+cleanup() {
+  if [[ -f /usr/share/pacstall/repo/pacstallrepo.pacstall-qa.bak ]]; then
+    echo "Returning '/usr/share/pacstall/repo/pacstallrepo' backup"
+    sudo rm -f /usr/share/pacstall/repo/pacstallrepo
+    sudo mv /usr/share/pacstall/repo/pacstallrepo.pacstall-qa.bak /usr/share/pacstall/repo/pacstallrepo
+  fi
+}
+trap cleanup EXIT
+trap cleanup INT
+/usr/share/pacstall-qa/pacstall-qa $@
```
Comment (Elsie19) [inline pacstall-qa.wrapper]:
```suggestion
/usr/share/pacstall-qa/pacstall-qa "$@"
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: Makefile [hunk 3]
```diff
@@ -1,4 +1,10 @@
...
 
 install:
-	install -Dmv755 pacstall-qa -t $(DESTDIR)/usr/bin
+	cp pacstall-qa.wrapper $(DESTDIR)/usr/bin/pacstall-qa
+	install -Dm755 pacstall-qa -t $(DESTDIR)/usr/share/pacstall-qa/
+	install -Dm644 README.md -t $(DESTDIR)/usr/share/pacstall-qa/
+
+uninstall:
+	rm /usr/bin/pacstall-qa
+	rm -r /usr/share/pacstall-qa
```
Comment (Elsie19) [inline Makefile:10]:
```suggestion
	rm -v /usr/bin/pacstall-qa
	rm -rfv /usr/share/pacstall-qa
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

**Review by Elsie19 — APPROVED** [review]

**Pull request merged**